tests/expected/lsfd/mkfds-tcp6

```diff
@@ -0,0 +1,4 @@
+    3 TCPv6   SOCK state=listen laddr=[::1]:34567                             listen    stream              1         ::1          :: [::1]:34567     34567      [::]:0         0
```

Comment:
Having the remote address being `[::]:0` instead of empty looks weird.
But it's the same for plain tcp.

Reply:
I'm not sure what the column (TCP.RADDR) should be for TCP listening socket.

In the NAME column, that is for human reading, lsfd hides raddr for such sockets:
```
# ./lsfd -Q '(SOCK.LISTENING && (TYPE == "TCP"))'       
COMMAND           PID            USER ASSOC MODE TYPE SOURCE MNTID INODE NAME
systemd             1            root   547  rw-  TCP sockfs     9  1327 state=listen laddr=0.0.0.0:111
systemd-resolve   981 systemd-resolve    11  rw-  TCP sockfs     9 20761 state=listen laddr=0.0.0.0:5355
...
```

ss command prints the an address:
```
ss -tl                                               
State                    Recv-Q                   Send-Q                                     Local Address:Port                                         Peer Address:Port                   Process                   
LISTEN                   0                        4096                                       127.0.0.53%lo:domain                                            0.0.0.0:*                                                
LISTEN                   0                        128                                            127.0.0.1:ipp                                               0.0.0.0:*                                                
LISTEN                   0                        4096                                          127.0.0.54:domain                                            0.0.0.0:*                                                
LISTEN                   0                        32                                         192.168.124.1:domain                                            0.0.0.0:*                                                
LISTEN                   0                        4096                                             0.0.0.0:hostmon                                           0.0.0.0:*                                                
LISTEN                   0                        128                                              0.0.0.0:ssh                                               0.0.0.0:*                                                
LISTEN                   0                        4096                                             0.0.0.0:sunrpc                                            0.0.0.0:*                                                
LISTEN                   0                        128                                                [::1]:ipp                                                  [::]:*                                                
LISTEN                   0                        4096                                                   *:websm                                                   *:*                                                
LISTEN                   0                        4096                                                [::]:hostmon                                              [::]:*                                                
LISTEN                   0                        128                                                 [::]:ssh                                                  [::]:*                                                
LISTEN                   0                        511                                                    *:http                                                    *:*                                                
LISTEN                   0                        4096                                                [::]:sunrpc                                               [::]:*                                                
LISTEN                   0                        1                                                  [::1]:italk                                                [::]:*                     
```

I don't have a strong option, but I cannot find a strong reason to empty the column.
@karelzak what do you think?

Reply:
We should provide optimized information for NAME, so skipping empty raddr= is fine. But for TCP.RADDR, it would be better to be explicit and provide empty addresses like `ss` (0.0.0.0,  [::]). I guess the port is displayed by TCP.RPORT.

Reply:
TCP.RADDR and TCP.RPORT are filled as you wrote as far as one of connect(2), bind(2), listen(2), or accept(2) is called on the socket. For the sockets just after being created with calling socket(2), the columns are not filled.